modules/phase_field/doc/content/source/kernels/ADACInterface.md

```diff
@@ -0,0 +1,21 @@
+# ADACInterface
+
+!syntax description /Kernels/ADACInterface<RESIDUAL>
+
+Implements the Allen-Cahn term for the $\frac{\kappa_i}2|\nabla \eta_i|^2$
+gradient energy contribution for the isotropic mobility case. Its weak form is
+
+\begin{equation}
+\left( \kappa_i \nabla \eta_i, \nabla \cdot (L_i \psi_m ) \right),
```

Comment:
I would remove the \cdot, since it is normally used to indicate divergence operator, but you are operating on a scalar not a vector

Reply:
done